dvc/info.py

```diff
@@ -47,6 +47,13 @@ def get_dvc_info():
             if psutil:
                 fs_type = get_fs_type(repo.cache.local.cache_dir)
                 info.append(f"Cache directory: {fs_type}")
+                external_cache = repo.cache.local.cache_dir
+                remote_cache = _get_external_cache(repo)
+                if external_cache:
+                    info.append(f"External file cache: {external_cache}")
+                if remote_cache:
+                    info.append(f"External remote cache: {remote_cache}")
```

Comment:
Seems like we can simply embed the cache dir into the `Cache directory` message above.

The remote caches are not dependant on `psutil`, so there is no reason to keep them under this `if`.

Reply:
Will cache_dir output the `external_cache_url` also? because i assumed it only outputs `directory_path` for caches

Reply:
In that particular case - yes, because we are talking about the local (external or not) cache. Again, I do understand that this might be a bit confusing. :slightly_frowning_face: 

So default, config's `cache.local` and `cache.dir` will be covered by `repo.cache.local.cache_dir` above. And the rest should be then standalone (e.g. config's `cache.s3`, `cache.ssh`, etc)

Reply:
Yeah i get you now😃